docs/erc1155-skills.md

```diff
@@ -0,0 +1,214 @@
+The AEA `erc1155_deploy` and `erc1155_client` skills demonstrate an interaction between two AEAs with the usage of a smart contract.
+
+* The `erc1155_deploy` skill deploys the smart contract, creates and mints items. 
+* The `erc1155_client` skill signs a transaction to complete a trustless trade with the counterparty.
+
+## Preparation instructions
+ 
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+##Discussion
+
+The scope of the specific demo is to demonstrate how to deploy a smart contract and interact with it. For the specific use-case, we create two AEAs one that deploys and creates tokens inside the smart contract and the other that signs a transaction so we can complete an atomic swap. The smart contract we are using is an ERC1155 smart contract
+with a one-step atomic swap functionality. That means the trade between the two AEAs can be trustless.
```

Comment:
We should add a `note` below saying:
`This demo serves demonstrative purposes only.  Since the AEA deploying the contract also has the ability to mint tokens, in reality the transfer of tokens from the AEA signing the transaction is pointless.`